test/remote/gateways/remote_paysafe_test.rb

```diff
@@ -0,0 +1,196 @@
+require 'test_helper'
+
+class RemotePaysafeTest < Test::Unit::TestCase
+  def setup
+    @gateway = PaysafeGateway.new(fixtures(:paysafe))
+
+    @amount = 100
+    @credit_card = credit_card('4107857757053670')
+    @pm_token = 'Ci3S9DWyOP9CiJ5'
+    @options = {
+      billing_address: {
+        address1: '999 This Way Lane',
+        city: 'Hereville',
+        state: 'NC',
+        country: 'FR',
+        zip: '98989',
+        phone: '999-9999999'
```

Comment:
Nothing wrong with defining this yourself, but you can just invoke an address via the `test_helper` by calling `address`.

Reply:
Good call - Revised!